designs/developer/add-45/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,89 @@
+
+* The Pipelines section allows the user to enable a pipeline template for their application, component or service.
+
+* A Pipelines section appears in +Add flows (for Import from Git, Developer Catalog, and From Dockerfile) when the Pipelines operator is installed.
+
+* When the Pipelines operator is not installed, this section is not displayed.
+
+## Import from Git
+### No Builder Image Selected
+* When no builder image or resource has been selected yet, the Pipelines section will display an info alert that says “Select a builder image and resource to see if there is a pipeline template available for this runtime.”
+
+![image](img/info-alert-1.png)
+
+### Builder Image Selected and Pipeline Template Exists
+* When a builder image has been selected, if a pipeline template is available, the Pipeline section shall be updated to display an “Add pipeline”  checkbox option.
+* The default state of the Add pipeline checkbox is OFF.
+* An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default.
+* A 'Dev Preview' label is featured next to 'Add pipeline' label.
+
+![img](img/import-from-git-devpreview-1.png)
+
+* After the user clicks the caret or 'Show pipeline visualization' label, the visualization is expanded and the user can hover over the different sections of the pipeline to see more details.  
+* The pipeline visualization label changes to ‘Hide pipeline visualization’ if the pipeline visualization is expanded.
+
+![image](img/import-from-git-devpreview-2.png)
+
+
+### Builder Image Selected and No Pipeline Template Exists
+* If a selected combination of a builder image and resource does not support a pipeline an info alert would appear alerting the user of this.
+* This message specifies the builder image and resource type that the user selected.
+
+![image](img/info-alert-2.png)
+
+
+### Developer Catalog
+
+#### Pipeline Template Exists
+* When an app, service, or source-to-image builder is added, if a pipeline template is available, the Pipeline section is updated to display an “Add pipeline”  checkbox option
+* The default state of the Add pipeline checkbox is OFF
+* An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_devpreview.png)
+
+#### No Pipeline Template Exists
+* When no pipeline template is available (which will be determined after the app, service, or source-to-image builder is added), an Info inline alert shall be displayed within the Pipeline section
+* The alert shall say “There are no pipeline templates available for this runtime.”
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_No_Temp.png)
```

Comment:
@lwrigh This image isn't loading for me.

Reply:
This is still broken it seems...

Reply:
Good catch! guessing you can't have a file start with "+"

Reply:
Got rid of it here as well and updated the markdown file.